testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCSdJwtIssuingEndpointTest.java

```diff
@@ -0,0 +1,397 @@
+                "sha-256",
+                "did:web:issuer.org",
+                2,
+                List.of(),
+                Optional.empty(),
+                VerifiableCredentialType.from("https://credentials.example.com/test-credential"),
+                CredentialConfigId.from("test-credential"));
+
+        SdJwtSigningService identityCredentialSigningService = new SdJwtSigningService(
+                session,
+                JsonSerialization.mapper,
+                getKeyFromSession(session).getKid(),
+                Algorithm.ES256,
+                Format.SD_JWT_VC.toString(),
+                "sha-256",
+                "did:web:issuer.org",
+                0,
+                List.of("given_name"),
+                Optional.empty(),
+                VerifiableCredentialType.from("https://credentials.example.com/identity_credential"),
+                CredentialConfigId.from("IdentityCredential"));
+
+        return new OID4VCIssuerEndpoint(
+                session,
+                "did:web:issuer.org",
+                Map.of(
+                        testCredentialSigningService.locator(), testCredentialSigningService,
+                        identityCredentialSigningService.locator(), identityCredentialSigningService
+                        ),
+                authenticator,
+                new ObjectMapper(),
+                TIME_PROVIDER,
+                30,
+                true);
+    }
+
+    private ComponentExportRepresentation getIdCredentialSigningProvider() {
+        ComponentExportRepresentation componentExportRepresentation = new ComponentExportRepresentation();
+        componentExportRepresentation.setName("sd-jwt-signing_identity_credential");
+        componentExportRepresentation.setId(UUID.randomUUID().toString());
+        componentExportRepresentation.setProviderId(Format.SD_JWT_VC.toString());
+
+        componentExportRepresentation.setConfig(new MultivaluedHashMap<>(
+                Map.of(
+                        "algorithmType", List.of("ES256"),
+                        "tokenType", List.of(Format.SD_JWT_VC.toString()),
+                        "issuerDid", List.of(TEST_DID.toString()),
+                        "hashAlgorithm", List.of("sha-256"),
+                        "decoys", List.of("0"),
+                        "vct", List.of("https://credentials.example.com/identity_credential"),
+                        "vcConfigId", List.of("IdentityCredential")
+                )
+        ));
+        return componentExportRepresentation;
+    }
+
+    private ComponentExportRepresentation getTestCredentialSigningProvider() {
+        ComponentExportRepresentation componentExportRepresentation = new ComponentExportRepresentation();
+        componentExportRepresentation.setName("sd-jwt-signing_test-credential");
+        componentExportRepresentation.setId(UUID.randomUUID().toString());
+        componentExportRepresentation.setProviderId(Format.SD_JWT_VC.toString());
```

Comment:
It might be `Format.SD_JWT_VC`

Reply:
Resolved with [d8cd042](https://github.com/keycloak/keycloak/pull/30692/commits/d8cd042dc629ebb73c665be5831483de8b9c2174)